ecp-utils.php

```diff
@@ -116,16 +101,13 @@ public function run_query_quick_address_scan( $current_time ) {
              LIMIT 1;"; // Try to use lower indexes first
 		$clean_address            = $wpdb->get_var( $query, 0, 0 );
 		$bch_cashaddr             = $wpdb->get_var( null, 1, 0 );
-
 		return array(
 			'btc_address'  => $clean_address,
 			'bch_cashaddr' => $bch_cashaddr,
 		);
 	}
 }
-
-class ElectrumBSVUtil extends ElectrumUtil {
-
+class ElectrumBTCUtil extends ElectrumUtil {
```

Comment:
this diff is weird. It's assuming you changed BSV to BTC and added BSV further down. 
Anyway to make this render in a nicer way to read?

Reply:
I did copy the ElectrumBSVUTIL section then pasted it just below. i then changed the original BSV section to BTC. I am sorry I dont know why there are extra lines I will try to edit..  I'm still learning git

Reply:
File updated I think it is right now